docs/faq.md

```diff
@@ -222,6 +226,10 @@ XMTP provides perceptibly real-time message delivery and retrieval. The network
 
 We don't provide a direct method to fetch all XMTP-enabled wallet addresses in bulk. However, you could consider utilizing a third-party service like Airstack. They have compiled an extensive list of XMTP addresses by querying our canMessage API over time at a rate that avoids hitting the limits. This approach could be a time-efficient alternative for you.
 
+## Group chats
```

Comment:
```suggestion
### Does XMTP support group chat?
```

Also may I suggest that we add this question to the Message patterns section? https://xmtp.org/docs/faq#message-patterns I think it would fit right in there. =)